exonum-java-binding/core/rust/exonum-java/Cargo.toml

```diff
@@ -5,15 +5,19 @@ authors = ["Exonum team <contact@exonum.com>"]
 edition = "2018"
 
 [dependencies]
+anyhow = "1.0"
 env_logger = "0.6"
-exonum-explorer-service = "1.0.0-rc.1"
-exonum-node = "1.0.0-rc.1"
-exonum-system-api = "1.0.0-rc.1"
-failure = "0.1"
+exonum-explorer-service = "1.0.0-rc.2"
+exonum-node = "1.0.0-rc.2"
+exonum-system-api = "1.0.0-rc.2"
 # Specify the "resource-manager" feature explicitly to prevent issues with linking (see ECR-2855).
 # Although its primary goal is to detect _programming_ errors that shall not occur
 # in a correct application early, it is currently required in the application
 # because some public APIs in Java permit passing arbitrary pointers (e.g., Snapshot#newInstance).
 java_bindings = { path = "..", features = ["resource-manager"] }
 log = "0.4"
 serde = { version = "1.0", features = ["serde_derive"] }
+
+[dependencies.tokio]
+version = "0.2.13"
+features = ["blocking", "dns", "io-util", "macros", "rt-threaded", "tcp", "time"]
```

Comment:
How are these features picked?

Reply:
It was actually copy-pasted from Core's cryptocurrency example... I'll check if we can get rid of them